pkg/grpc/universalapi/api_workflow.go

```diff
@@ -125,30 +125,30 @@ func (a *UniversalAPI) TerminateWorkflowAlpha1(ctx context.Context, in *runtimev
 	return a.workflowActivity(ctx, in, method, messages.ErrTerminateWorkflow)
 }
 
-func (a *UniversalAPI) RaiseEventWorkflowAlpha1(ctx context.Context, in *runtimev1pb.RaiseEventWorkflowRequest) (*runtimev1pb.RaiseEventWorkflowResponse, error) {
+func (a *UniversalAPI) RaiseEventWorkflowAlpha1(ctx context.Context, in *runtimev1pb.RaiseEventWorkflowRequest) (*runtimev1pb.WorkflowActivityResponse, error) {
```

Comment:
I suggest reverting these changes because I'm making additional changes to these signatures in a nother PR.